.github/workflows/ci-build-spm.yml

```diff
@@ -40,9 +48,15 @@ jobs:
       uses: ./.github/actions/setup-node.js
 
     - name: Temporary - only run the build
-      run:
-        cd docker-compose/monitor && make build
+      run: |
+        case ${{ matrix.mode.name }} in
+          v1)
+            cd docker-compose/monitor && make build
```

Comment:
looks like you can simply do `make build-${{ matrix.mode.name }}`. Rename `build` target to `build-v1` and add an alias `build -> build-v1`